routes/accounts.js

```diff
@@ -17,14 +15,8 @@ router.get( '/register', ( request, response ) => {
   response.render( 'accounts/register' )
 })
 
-router.post( '/register', ( request, response , next) => {
-  const User = request.app.get( 'models' ).User
-
-  const { email, password } = request.body
-
-  User.create({ email, password: encryptPassword( password ) })
-    .then( user => RegistrationEmail.send( user ))
-    .then( user => response.redirect( '/accounts/verify' ))
+router.post( '/register', ( request, response ) => {
+  register( request, response.redirect( '/accounts/verify' ) )
```

Comment:
This code will *always* redirect to the `/accounts/verify` endpoint - I think you intended:
```js
register( request, response )
```